services/web-app/layouts/layout/layout.js

```diff
@@ -83,14 +83,28 @@ const Layout = ({ children }) => {
                     className={styles.headerNavigation}
                   >
                     {globalNavData.map((data) => (
-                      <HeaderMenuItem
-                        key={data.path}
-                        isCurrentPage={router.pathname.startsWith(data.path)}
-                        href={data.path}
-                        element={NextLink}
-                      >
-                        {data.title}
-                      </HeaderMenuItem>
+                      <>
+                        {data.path && (
+                          <HeaderMenuItem
+                            key={data.title}
+                            isCurrentPage={data.path && router.pathname.startsWith(data.path)}
+                            href={data.path && data.path}
+                            element={data.path && NextLink}
+                            tabIndex={!data.path && '-1'}
+                          >
```

Comment:
```suggestion
                          <HeaderMenuItem
                            key={data.title}
                            isCurrentPage={router.pathname.startsWith(data.path)}
                            href={data.path}
                            element={NextLink}
                          >
```
At this point we already checked data.path exists in line 87, are these checks still necessary?

Reply:
Changing this might bring the CC down, but let me know if you want to pair up on that (https://sonarcloud.io/project/issues?resolved=false&types=CODE_SMELL&pullRequest=564&id=carbon-design-system_carbon-platform&open=AYApPXRdF3rg56pYIdKX)

Reply:
oops, I forgot to remove that. good catch